exponent2.c

```diff
@@ -0,0 +1,26 @@
+//inefficient
```

Comment:
also more comments! you're right that it could have implemented more efficiently, but still solid job!